client/src/app/modules/auth/confirm-email/confirm-email.component.ts

```diff
@@ -27,10 +28,12 @@ export class ConfirmEmailComponent implements OnInit {
     this.activatedRoute.queryParams.subscribe((params) => {
       const confirmEmailToken = params['token'];
 
-      this.authService.confirmEmail({ confirmEmailToken }).subscribe(() => {
-        clearInterval(dotsInterval);
-        this.router.navigateByUrl('auth/sign-in');
-      });
+      this.authService.confirmEmail({ confirmEmailToken })
+        .pipe(delay(3000))
+        .subscribe(() => {
+          clearInterval(dotsInterval);
```

Comment:
I think it's better to `clearInterval` on component destroy, so then we don't have to wait 3 sec